main.py

```diff
@@ -1,59 +1,82 @@
 import labelbox
 from labelbox import Client, Dataset
 from labelbox.schema.data_row_metadata import DataRowMetadata, DataRowMetadataField, DataRowMetadataKind, DeleteDataRowMetadata
+from processing.metadata import *
 import os
 
-# Add your API key below
-labelbox_api_key = os.environ.get("labelbox_api_key", '')
-try:
-    client = Client(api_key=labelbox_api_key)
-except:
-    print(f'Could not connect to Labelbox. Please provide the Cloud Function with a Labelbox API KEY as an Environment Variable with the name "labelbox_api_key"')
-
-
 def stream_data_rows(event, context):
-    """ Uploads an asset to Catalog when a new asset is uploaded to GCP bucket. Creates a dataset if a dataset with this bucket name doesn't exist yet
+    """ 
```

Comment:
Multiple new lines here may throw off the autocomplete functions of some editors.